src/components/Staking/Overview/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect } from 'react';
```

Comment:
Note to myself, since 'normal' staking code was reorganized, this too will need to be tested

Reply:
Alright :-)